src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodEnvVar.java

```diff
@@ -94,4 +93,14 @@ public String getDisplayName() {
             return "Global Environment Variable (applied to all containers)";
         }
     }
+
+    public interface Names {
```

Comment:
I named it so because it's a part of PodEnvVar, so PodEnvVar.Names make sense alright.
In places where it's used I use full import, so it's not confusing. But I agree, full name will still make better sense.